src/components/gas/GasSpeedButton.tsx

```diff
@@ -348,21 +346,13 @@ const GasSpeedButton = ({
 
   const openGasHelper = useCallback(async () => {
     Keyboard.dismiss();
-    if (crossChainServiceTime) {
```

Comment:
could we remove `crossChainServiceTime` as a prop from `GasSpeedButton` - also removes the logic at line 316 and would need to remove from `SignTransactionSheet` and `SendConfirmationSheet`